Tasks/FtpUpload/task.json

```diff
@@ -0,0 +1,97 @@
+{
+  "id": "6f8c69a5-b023-428e-a125-fccf4efcb929",
+  "name": "FtpUpload",
+  "friendlyName": "Ftp Upload",
+  "description": "Ftp Upload",
```

Comment:
Description:
Upload files to a remote machine using the File Transfer Protocol (FTP), or securely with FTPS.